Framework/Kernel/inc/MantidKernel/PropertyWithValue.tcc

```diff
@@ -399,13 +399,13 @@ void PropertyWithValue<TYPE>::replaceValidator(IValidator_sptr newValidator) {
 template <typename TYPE>
 std::string
 PropertyWithValue<TYPE>::setValueFromProperty(const Property &right) {
-  
```

Comment:
I think merging this change will break clangformat on `master` as the `.tcc` files are now included and this will fail. Can you rebase with `master` and remove the change to this file?